### ISSUE TITLE
Finetuning with proposals from GA RPN

### ISSUE THREAD

**Issue by sehyungp92** [open-mmlab/mmdetection#3226]
In the paper for HTC, it states that the trained model was finetuned using high quality pre-computed proposals from the GA RPN. I wanted to explore this approach to improve my results as well and just wanted to find out what steps I would have to take to do so (e.g. how to use pre-computed proposals for models like HTC). Any guidance would be much appreciated.

Many thanks.

**Comment by ZwwWayne** [comment]
Please use the `Reimplementation Questions` when creating an issue for such questions.

**Comment by sehyungp92** [comment]
Thanks, I'll create a new post with the reimplementation label.
